lib/fastlane/plugin/wpmreleasetoolkit/actions/common/comment_on_pr.rb

```diff
@@ -0,0 +1,74 @@
+require 'fastlane/action'
+require 'rubygems/command_manager'
+require 'securerandom'
+
+module Fastlane
+  module Actions
+    class CommentOnPrAction < Action
+      def self.run(params)
+        require_relative '../../helper/github_helper'
+
+        Fastlane::Helper::GithubHelper.comment_on_pr(
+          project_slug: params[:project],
+          pr_number: params[:pr_number],
+          body: params[:body],
+          reuse_identifier: params[:reuse_identifier]
+        )
+      end
+
+      def self.description
+        'Post a comment on a given PR number (optionally updating an existing one)'
+      end
+
+      def self.authors
+        ['Jeremy Massel']
```

Comment:
Not: I thought we said that from now on we'd use "Automattic" s the author of actions?

Reply:
I copy/pasted from an old action and saw there was a name in it so I didn't know if people were doing this or not.

I've updated it to be `Automattic`, because I actually prefer it that way 😁